src/io/output_mesh.cc

```diff
@@ -514,6 +547,12 @@ void OutputMeshDiscontinuous::refine_aux_element(const OutputMeshDiscontinuous::
     }
 }
 
+void OutputMeshDiscontinuous::create_sub_mesh()
```

Comment:
We should make a method that constructs discont mesh from continuous in order to have just one